src/app/components/MediaLoader/index.styles.ts

```diff
@@ -52,21 +55,38 @@ export default {
       !isEmbedded && commonMarginSpacing,
     ],
 
-  audioMediaContainer: ({ palette }: Theme) =>
+  audioMediaContainer: ({ mq, spacings }: Theme) =>
     css({
-      backgroundColor: palette.WHITE,
       height: '165px',
+      '.audio-post &&': {
+        height: '130px',
+        margin: `0 0 ${spacings.FULL}rem 0`,
+        [mq.GROUP_2_MIN_WIDTH]: {
+          padding: `0 ${spacings.FULL}rem`,
+        },
+        [`@media (min-width: 548px)`]: {
+          padding: '0 0',
+        },
+        [mq.GROUP_3_MIN_WIDTH]: {
+          marginInlineEnd: `${spacings.DOUBLE}rem`,
+        },
+        [mq.GROUP_4_MIN_WIDTH]: {
+          marginInlineStart: `-${spacings.DOUBLE}rem`,
+        },
```

Comment:
OR, given that this is for live posts with audio player, do we need to move this to the Post styles file?

Reply:
When I try to move it, the styling of the caption goes awry 😬